webpack/components/ReportJsonViewer.js

```diff
@@ -10,7 +10,7 @@ const theme = {
 
 const ReportJsonViewer = ({ data }) => (
   <div className="report-json-viewer">
-    <JSONTree data={this.props.data} hideRoot theme={theme} />
+    <JSONTree data={data} hideRoot theme={theme} />
```

Comment:
is this fix related?

Reply:
Yes, report is not displayed properly without it:

![data-without](https://user-images.githubusercontent.com/2664090/89033165-7f8f1680-d336-11ea-8cf1-991f282693b1.png)

![data-within](https://user-images.githubusercontent.com/2664090/89033168-8158da00-d336-11ea-8513-e9b46fc109cd.png)

Reply:
Thanks for explanation :+1: